com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -769,7 +769,7 @@ private unsafe void SendParametersUpdateServerRpc(ParametersUpdateMessage parame
                     return;
                 }
                 UpdateParameters(parametersUpdate);
-                if (NetworkManager.ConnectedClientsIds.Count - 2 > 0)
+                if (IsHost && NetworkManager.ConnectedClientsIds.Count > 2 || NetworkManager.ConnectedClientsIds.Count > 1)
```

Comment:
I see...yes that is a good catch indeed!
How about we condense that to something like:
```
                if (NetworkManager.ConnectedClientsIds.Count > (IsHost ? 2 : 1) )
                {
```
Thank you for your contribution and taking the time to troubleshoot that!  👍

Reply:
Can do, but isn't it more clear as it is?

Reply:
Well...with your current suggested fix you could still have a Host with one connected client and it would pass the conditional "If" check.  My update to your suggestion assures that if it is a Host it requires more than 2 connected clients, otherwise if it is server it only requires one.
Does that make sense?

Reply:
Indeed, we overlocked it

Reply:
Done.